Source/Parsers/Parser Extensions/SVGKParserGradient.m

```diff
@@ -16,24 +16,25 @@
 #import "SVGLinearGradientElement.h"
 #import "SVGRadialGradientElement.h"
 
+@interface SVGKParserGradient ()
+@property (nonatomic) NSArray *supportedNamespaces;
+@property (nonatomic) NSArray *supportedTags;
+@end
+
 @implementation SVGKParserGradient
 
-//-(NSSet*) supportedNamespaces
```

Comment:
This required protocol method does not implements. And, all the following return value should be `NSArray`, but not `NSSet`. See the protocol definition.